examples/vlexamples.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+// jshint quotmark: false
+
+var VL_SPECS = {
+  'Basic': [
+    {
+      "name": "bar",
+      "title": "Simple Bar Chart"
```

Comment:
That's a good question because I don't know if we should add "title" as top-level property.  
(Maybe we do, and let's support #431?)   That said, this might has some conflict with the vega-embed spec format.